includes/Config/DataAccountingConfig.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace DataAccounting\Config;
+
+use Config;
+use Exception;
+use MultiConfig;
+use MutableConfig;
+
+class DataAccountingConfig extends MultiConfig implements MutableConfig {
+	/**
+	 * @var Handler 
+	 */
+	protected $handler = null;
```

Comment:
@pwirth while fixing psalm type annotation stuff, it complains here that
```
ERROR: PossiblyNullPropertyAssignmentValue - includes/Config/DataAccountingConfig.php:14:23 - $this->handler with non-nullable declared type 'DataAccounting\Config\Handler' cannot be assigned nullable type 'null' (see https://psalm.dev/148)
```

Can I safely delete the `= null` here?

Reply:
OK, I found that it is not possible. Should the type be `Handler?` then?